### DISCUSSION TITLE
Share an underrated repository which deserves 1,000 stars

### DISCUSSION THREAD

**Discussion by NitinIsFit** [community/community#27260] [Other Feature Feedback, Questions, & Ideas]
For me it's [hazy](https://github.com/Nit-nit/hazy)-my discord bot

**Comment by aayushchugh** [comment]
https://github.com/shriproperty/

**Reply by NitinIsFit** [reply]
Real estate, coool

**Reply by aayushchugh** [reply]
can you mark this answer I might get new badge on my profile
thanks

**Reply by NitinIsFit** [reply]
Ok lol, but i will mark the most upvoted answer later

**Reply by aayushchugh** [reply]
no problem

**Reply by NitinIsFit** [reply]
Thanks for understanding
